llvm/lib/ProfileData/MemProfReader.cpp

```diff
@@ -610,13 +670,33 @@ RawMemProfReader::peekBuildIds(MemoryBuffer *DataBuffer) {
   return BuildIds.takeVector();
 }
 
+// FIXME: Add a schema for serializing similiar to IndexedMemprofReader. This
+// will help being able to deserialize different versions raw memprof versions
+// more easily.
+llvm::SmallVector<std::pair<uint64_t, MemInfoBlock>>
+RawMemProfReader::readMemInfoBlocks(const char *Ptr) {
+  if (MemprofRawVersion == 3ULL) {
+    errs() << "Reading V3\n";
```

Comment:
Leftover debugging message that should be removed?

Reply:
Once you do that, the braces can be removed from all of the if else conditions here which have single statement bodies